TopAnalysis/data/grouping/fcncTrilepton.yaml

```diff
@@ -0,0 +1,100 @@
+processes:
+    DataElEl:
+        title: "Data"
+        datasets:
+            - Run2016.DoubleEG
+            - Run2016.SingleElectron
+    DataMuMu:
+        title: "Data"
+        datasets:
+            - Run2016.DoubleMuon
+            - Run2016.SingleMuon
+    DataMuEl:
+        title: "Data"
+        datasets:
+            - Run2016.MuonEG
+            - Run2016.SingleMuon
+            - Run2016.SingleElectron
+
+    DYJets:
+        title: "Z/#gamma^{*}+Jets#rightarrow l^{+}l^{-}"
+        weight: "puWeight*BtagWeight"
+        #weight: "genWeight*puWeight*BtagWeight"
```

Comment:
please use the "genWeight*puWeight*BtagWeight"

Reply:
Do I use genWeight only at fcnc? or ttbb together?

Reply:
better together.

Reply:
Ok, then I will do that with ttbb. Thanks!